### ISSUE TITLE
appconfiguration@2022-05-01 - POST method `checkNameAvailability` is not generated 

### ISSUE THREAD

**Issue by teowa** [hashicorp/pandora#1373]
This POST method [`checkNameAvailability`](https://github.com/Azure/azure-rest-api-specs/blob/25fc6633c6feecfc01fd46f4926947f31d2e5945/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2022-05-01/appconfiguration.json#L1150-L1157) is not in the generted SDK [appconfiguration 2022-05-01](https://github.com/hashicorp/go-azure-sdk/tree/main/resource-manager/appconfiguration/2022-05-01).

This method is needed in PR https://github.com/hashicorp/terraform-provider-azurerm/pull/17714#issuecomment-1225449556. 

[Swagger](https://github.com/Azure/azure-rest-api-specs/blob/25fc6633c6feecfc01fd46f4926947f31d2e5945/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2022-05-01/appconfiguration.json#L1150-L1157):
```
    "/subscriptions/{subscriptionId}/providers/Microsoft.AppConfiguration/locations/{location}/checkNameAvailability": {
      "post": {
        "tags": [
          "Operations"
        ],
        "description": "Checks whether the configuration store name is available for use.",
        "operationId": "Operations_RegionalCheckNameAvailability",
        "parameters": [
```
Maybe there is something special with the tag `Operations`.

**Comment by teowa** [comment]
The method with tags `Operations` is intentionally filtered by https://github.com/hashicorp/pandora/blob/416bcc5037ccfbbb0c03084a78cd40f9e2ee928e/tools/importer-rest-api-specs/components/parser/swagger_tags.go#L10

**Comment by stephybun** [comment]
Closed by #2363
